TwitchDownloaderWPF/Translations/Strings.zh-cn.resx

```diff
@@ -934,78 +934,78 @@
     <value>使用精确模式修剪的视频在前几秒或最后几秒可能会出现视频 / 音频卡顿。安全模式保证不会出现卡顿，但可能会导致视频稍长。</value>
   </data>
   <data name="TaskRetry" xml:space="preserve">
-    <value>Retry</value>
+    <value>重试</value>
   </data>
   <data name="UnableToGetChannelVideos" xml:space="preserve">
-    <value>Unable to Get Videos</value>
+    <value>无法获取视频</value>
   </data>
   <data name="UnableToGetChannelVideosMessage" xml:space="preserve">
-    <value>Unable to get channel videos: {0}</value>
+    <value>无法获取频道视频：{0}</value>
   </data>
   <data name="UnableToGetChannelClips" xml:space="preserve">
-    <value>Unable to Get Clips</value>
+    <value>无法获取剪辑</value>
   </data>
   <data name="UnableToGetChannelClipsMessage" xml:space="preserve">
-    <value>Unable to get channel clips: {0}</value>
+    <value>无法获取频道剪辑：{0}</value>
   </data>
   <data name="FilenameParameterTitleTooltip" xml:space="preserve">
-    <value>The title of the video/clip.</value>
+    <value>视频 / 剪辑的标题。</value>
   </data>
   <data name="FilenameParameterIdTooltip" xml:space="preserve">
-    <value>The ID of the video/clip.</value>
+    <value>视频 / 剪辑的 ID。</value>
   </data>
   <data name="FilenameParameterDateTooltip" xml:space="preserve">
-    <value>The date that the video/clip was created in the format M-d-yy.</value>
+    <value>视频 / 剪辑的创建日期，格式为 M-d-yy。</value>
   </data>
   <data name="FilenameParameterDateCustomTooltip" xml:space="preserve">
-    <value>The date that the video/clip was created in a customizable format.</value>
+    <value>视频 / 剪辑的创建日期，格式为自定义格式。</value>
   </data>
   <data name="FilenameParameterChannelTooltip" xml:space="preserve">
-    <value>The display name of the channel which owns the video/clip/chat.</value>
+    <value>拥有此视频 / 剪辑 / 聊天的频道的显示名称。</value>
   </data>
   <data name="FilenameParameterRandomStringTooltip" xml:space="preserve">
-    <value>A string of 11 random characters.</value>
+    <value>由 11 个随机字符组成的字符串。</value>
   </data>
   <data name="FilenameParameterTrimStartTooltip" xml:space="preserve">
-    <value>The start trim of the video/chat in the format hh-mm-ss.</value>
+    <value>视频 / 聊天的修剪开始时间，格式为 HH-mm-ss。</value>
   </data>
   <data name="FilenameParameterTrimStartCustomTooltip" xml:space="preserve">
-    <value>The start trim of the video/chat in a customizable format.</value>
+    <value>视频 / 聊天的修剪开始时间，格式为自定义格式。</value>
   </data>
   <data name="FilenameParameterTrimEndTooltip" xml:space="preserve">
-    <value>The end trim of the video/chat in the format hh-mm-ss.</value>
+    <value>视频 / 聊天的修剪结束时间，格式为 HH-mm-ss。</value>
```

Comment:
The `hh` being lowercase instead of uppercase is intentional. `HH` is not a valid timespan format, but `hh` is.